index.js

```diff
@@ -116,12 +116,93 @@ LocationForecast.prototype = {
       if (err) {
         return callback(err, null);
       }
-      
+
       return callback(null, res);
     });
   },
 
 
+    /**
+     * Get seven day weather.
+     * @param {Function} callback
+     */
+    getSevenDaySummary: function(callback) {
```

Comment:
How about instead of these two functions you just created a single function named something like `getWeatherForNextDays(days, callback)`. This could allow you to pass `9` or `7` like so:

```js
weather.getWeatherForNextDays(7, function(err, summary) {
  console.log(err, summary)
});
```

I think this makes more sense since it can take any range for developers who need maybe just 2 or 3 days.